components/ui/pathogen-map/pathogen-map-layer.tsx

```diff
@@ -59,13 +44,41 @@ export function PathogenMapLayer<
   };
 
   return (
-    <Source id={sourceId} type="geojson" data={geojsonData}>
-      <Layer
-        id={layer.id}
-        type="circle"
-        source={sourceId}
-        paint={layer.layerPaint}
-      />
+    <Source
+      id={sourceId}
+      type="geojson"
+      data={geojsonData}
+      cluster
+      clusterMaxZoom={6}
+      clusterRadius={100}
+      clusterProperties={clusterProperties}
+    >
+      {layers.map((layer) => {
+        return layer.type == "symbol" ? (
+          <Layer
+            key={layer.id}
+            id={layer.id}
+            type="symbol"
```

Comment:
How does `type: "symbol"` reach this portion of the code? I don't understand 100%, is this unused code?